net/HttpRequest.hpp

```diff
@@ -1361,6 +1361,18 @@ class Session final : public ProtocolHandlerInterface
 #endif
     }
 
+    int getSslVerifyResult()
```

Comment:
The signature on `Socket::` is `long getSslVerifyResult()`. So this should match.
`int` and `long` are two different types.

Reply:
Also `_handshakeSslVerifyFailure` is a long

Reply:
```suggestion
    long getSslVerifyResult()
```

Reply:
Well spotted, missed it.

It seems we don't use -Werror :/ I imagine this would have been spotted then.

Reply:
`-Werror` wouldn't do anything if there is no warning emitted.